src/index.js

```diff
@@ -1,4 +1,168 @@
 'use strict'
 
-exports.IPLDService = require('./ipld-service')
-exports.resolve = require('./resolve')
+const Block = require('ipfs-block')
+const pull = require('pull-stream')
+const traverse = require('pull-traverse')
+const utils = require('./utils')
+const CID = require('cids')
+const until = require('async/until')
+const IPFSRepo = require('ipfs-repo')
+const MemoryStore = require('../node_modules/interface-pull-blob-store/lib/reference.js')
```

Comment:
right! (I do remember fighting with this one and this was how I got it to reference to the right one, not sure what was the error without node_modules though).

Could you also check https://github.com/ipfs/interface-pull-blob-store/pull/6, this way we don't even have to point to a path inside.